crates/snapbox/src/lib.rs

```diff
@@ -244,3 +244,14 @@ pub fn assert_subset_matches(
         .action_env(DEFAULT_ACTION_ENV)
         .subset_matches(pattern_root, actual_root);
 }
+
+/// A [`std::assert_eq`] compatible wrapper around the [`Assert`] API.
+#[macro_export]
+macro_rules! assert_eq {
```

Comment:
I think it would make sense to also add respective macros for `assert_matches` and `assert_eq_path` and `assert_matches_path` ... however I *really* dislike the names of the latter two, see #225. So I'd rather have us do a breaking rename before adding the new same-named macros.